sqlx-core/src/from_row.rs

```diff
@@ -92,6 +92,37 @@ use crate::row::Row;
 /// will set the value of the field `location` to the default value of `Option<String>`,
 /// which is `None`.
 ///
+/// ## Manual implementation
+///
+/// You can also implement the [`FromRow`] trait by hand. This can be useful if you
+/// have a struct with a field that needs manuel decoding:
+///
+/// ```rust
+/// # extern crate sqlx;
+/// use sqlx::FromRow;
+/// use sqlx::error::Result;
```

Comment:
We prefer that people use `sqlx::Result<...>` to make it clearer why `Result` only has one type parameter when you'd normally expect to see two.